packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -538,6 +560,7 @@ class CircularProgressIndicator extends ProgressIndicator {
     super.color,
     super.valueColor,
     this.strokeWidth = 4.0,
+    this.strokeAlign = BorderSide.strokeAlignCenter,
```

Comment:
I can copy these constants from BorderSide if it can make the code better, but reuse then seem applicable to me.